tests/bsim/bluetooth/audio_samples/broadcast_audio_sink/prj.conf

```diff
@@ -0,0 +1,20 @@
+# This file content is just a copy of the broadcast audio sink sample prj.conf
+
+CONFIG_BT=y
+CONFIG_LOG=y
+CONFIG_BT_AUDIO=y
+CONFIG_BT_SMP=y
+CONFIG_BT_PAC_SNK=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_BAP_BROADCAST_SINK=y
+CONFIG_BT_BAP_SCAN_DELEGATOR=y
+CONFIG_BT_ISO_MAX_CHAN=2
+CONFIG_BT_BAP_BROADCAST_SNK_SUBGROUP_COUNT=2
+CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT=2
+CONFIG_BT_BAP_SCAN_DELEGATOR_MAX_SUBGROUPS=2
+CONFIG_BT_BUF_ACL_RX_SIZE=255
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+
+CONFIG_BT_DEVICE_NAME="Broadcast Audio Sink"
+
+CONFIG_BT_TINYCRYPT_ECC=y
```

Comment:
I know there is an issue with the board files, but wouldn't it be possible to just have this as just `CONFIG_BT=y` and then compile the test with the `prj.conf` from the sample as an overlay file?

Reply:
We can point to the sample prj.conf altogether directly, and leave this as just an "empty" file telling people to use the other. It will be a bit less evident for those looking, but can be done.

Reply:
I'm 90% sure that if anyone besides you or me modifies the sample prj.conf, those changes will not make it here. 

So if we can rely on the _actual_ sample prj.conf, rather than assuming that we can manually keep these synced (and they shall be synced for the sample to actually be tested), then I think that would be a better solution.

Reply:
done